scripts/runfabtests.sh

```diff
@@ -284,11 +290,60 @@ function cs_test {
 	fi
 }
 
+function complex_test {
+	local test=$1
+	local config=$2
+	local test_exe="fi_${test}"
+	local start_time
+	local end_time
+	local test_time
+
+	# TODO: Add a better way to get to $datarootdir
+	if [ ! -f "${BIN_PATH}../share/fabtests/test_configs/${PROV}/${config}.json" ]; then
```

Comment:
Doesn't the ubertest know the default path after it is installed?  I would have it use the default search path, if not specified -- which it wouldn't be when run from the script, and fail gracefully if the file cannot be opened.